dev/com.ibm.ws.jaxrs.2.0.ejb/src/com/ibm/ws/jaxrs20/ejb/JaxRsFactoryBeanEJBCustomizer.java

```diff
@@ -244,6 +244,7 @@ public boolean isCustomizableBean(Class<?> clazz, Object context) {
     }
 
     @Override
+    @FFDCIgnore(Exception.class)
```

Comment:
```suggestion
    @FFDCIgnore({NamingException.class, Exception.class, Exception.class})
```

The `@FFDCIgnore` annotation scans through the method and looks for catch blocks based on the exception(s) in the value of the annotation.  It ignores based on matches in order, so for this method, it would only ignore the first catch block where it specifically catches an `Exception`.  This method has three catch blocks, first a `NamingException`, then two for `Exception`.  The FFDC reported by the user was from the last catch block, but we should probably ignore exceptions for all three - in the first catch block, we log the exception to trace, and in the second and third we re-throw the exception so it will be handled further upstream.